runtime/moonbase/src/lib.rs

```diff
@@ -601,16 +627,17 @@ impl pallet_ethereum::Config for Runtime {
 
 parameter_types! {
 	pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
+	pub const ReservedDmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
 }
 
 impl cumulus_pallet_parachain_system::Config for Runtime {
 	type Event = Event;
 	type OnValidationData = ();
-	type SelfParaId = ParachainInfo;
-	type DmpMessageHandler = ();
-	type ReservedDmpWeight = ();
-	type OutboundXcmpMessageSource = ();
-	type XcmpMessageHandler = ();
+	type SelfParaId = parachain_info::Pallet<Runtime>;
```

Comment:
```suggestion
	type SelfParaId = ParachainInfo;
```
not sure why this would've had to change